lib/grandpa/message.go

```diff
@@ -56,7 +56,11 @@ type VoteMessage struct {
 	Message SignedMessage
 }
 
-// Index returns VDT index
+func (v VoteMessage) String() string {
+	return fmt.Sprintf("round=%d, setID=%d, message={%s}", v.Round, v.SetID, v.Message)
+}
+
+// Index Returns VDT index
```

Comment:
nit to revert the not-so-needed delta

```suggestion
// Index returns VDT index
```